tests/offload_utils.hpp

```diff
@@ -15,7 +15,11 @@
 //@HEADER
 
 #ifdef _MDSPAN_HAS_SYCL
+#if __has_include(<sycl/sycl.hpp>)
 #include <sycl/sycl.hpp>
+#else
+#include <CL/sycl.hpp>
```

Comment:
Perhaps this SYCL implementation does not provides this deprecated SYCL 1.2.1 header now?

Reply:
Yeah, this one only had the deprecated header. Not quite sure if updating the compiler in the CI is worth it for this pull request.